OracleSiebel/OEL/build/multi-stage/cgw/cgw_install_base.sh

```diff
@@ -0,0 +1,19 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Author: andreas.thieltges@oracle.com
+# Description: Install Siebel IP17 Base for CGW
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+# #DF# Double check this is required.
+
+chown -R siebel /mnt/Siebel_Enterprise_Server
```

Comment:
These four commands can be combined into a single command:
```
chown siebel:siebel /mnt/Siebel_Enterprise_Server siebel
```

Reply:
Done. Thanks